homeassistant/components/home_connect/api.py

```diff
@@ -211,14 +213,34 @@ def get_opstate_sensor(self):
 class DeviceWithDoor(HomeConnectDevice):
     """Device that has a door sensor."""
 
-    def get_door_entity(self):
+    def get_door_entities(self):
         """Get a dictionary with info about the door binary sensor."""
-        return {
-            ATTR_DEVICE: self,
-            ATTR_DESC: "Door",
-            ATTR_SENSOR_TYPE: "door",
-            ATTR_DEVICE_CLASS: "door",
-        }
+        entities = []
+        status_keys = self.appliance.get_status().keys()
+        for key in status_keys:
```

Comment:
Not reusing status_keys - just directly assign:
`for key in self.appliance.get_status().keys()`

Reply:
Tried to change this but ruff seems to prefer a var here:
```
homeassistant/components/home_connect/api.py:219:13: SIM118 Use `key in dict` instead of `key in dict.keys()`
```

Reply:
I think it's just saying to use this instead:
`for key in self.appliance.get_status()`

Reply:
Ah, thanks. Updated in https://github.com/home-assistant/core/pull/102753/commits/8474bf55ac4c385836fa81c247b736b8aae0fb33.